packages/component/src/Styles/StyleSet/Tooltip.ts

```diff
@@ -0,0 +1,62 @@
+export default () => ({
+  '*:has(> &.webchat__tooltip)': {
```

Comment:
Do we have a test dedicated for white-label?

Reply:
Yes, added